ipa/prover.go

```diff
@@ -17,6 +17,17 @@ func init() {
 	maxEvalPointInsideDomain.SetUint64(common.VectorLength - 1)
 }
 
+var (
+	labelDomainSep   = []byte("ipa")
+	labelC           = []byte("C")
+	labelInputPoint  = []byte("input point")
+	labelOutputPoint = []byte("output point")
+	labelW           = []byte("w")
+	labelL           = []byte("L")
+	labelR           = []byte("R")
+	labelX           = []byte("x")
```

Comment:
Not needed in this PR, we should probably doc comment what these labels are used for in the protocol if they are now globals

Reply:
Note: they're unexported globals, and that's very important for security.

I'll do it in this PR; I'll add some comments with some pseudo-code description on how each label is used in the (non-)interactive protocol.

Reply:
Done, see new review!